exchanges/orders.py

```diff
@@ -0,0 +1,146 @@
+from .assets import Asset
+
+def basic_order_check(action, asset_pair):
+    if action not in ['buy', 'sell']:
+        raise ValueError("Action must be 'buy' or 'sell'")
+
+    asset0, asset1 = asset_pair
+    if not isinstance(asset0, Asset) or not isinstance(asset1, Asset):
+        raise ValueError("Not recognized asset pair")
+
+
+class Order(object):
+    """This is the simplest type of order, given an action (i.e. BUY/SELL);
+       an assets_pair (A0, A1) and a volume, it represents:
+
+       BUY/SELL `volume` of A0 using/for some volume of A1, which follows the
+       relationship: volume(A0)*price = volume(A1). Where `price` is dictated
+       by the market
+
+       Examples
+       --------
+       Order(action:BUY, asset_pair:(ETH, BTC), volume:1.5) means:
+       "BUY 1.5 Ethereum using as much Bitcoin as the market dictates"
+
+       Order(action:SELL, asset_pair:(Bitcoin, Ethereum), volume:1.5) means:
+       'SELL 1.5 Ethereum for the amount of Bitcoin the market dictates'"""
+    def __init__(self, action, asset_pair, volume,
+                 leverage=None, only_validate=False):
+        basic_order_check(action, asset_pair)
+        self._action = action
+        self._asset_pair = asset_pair
+        self._volume = volume
+        self._leverage = leverage
+        self._validate = only_validate
+
+    @property
+    def action(self):
+        return self._action
+
+    @property
+    def asset_pair(self):
+        return self._asset_pair
+
+    @property
+    def volume(self):
+        return self._volume
+
+    @property
+    def leverage(self):
+        return self._leverage
+
+    @property
+    def validate(self):
```

Comment:
Using the verb validate for this is confusing, it sounds like you want to validate the order. Maybe `is_mock`?